.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,60 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ main ]
+  schedule:
+    - cron: '17 23 * * 3'
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
```

Comment:
Just Question: What is the role of the setting `security-events`? If we do not permit the `write`, what happens?

Reply:
The reason for setting `security-event` to write permission is probably to POST the result to a specific location on the Security tab.

https://github.com/tier4/obstacle_stop_planner_refine/security/code-scanning?query=ref%3Arefs%2Fpull%2F38%2Fmerge

Reference:
https://docs.github.com/ja/actions/reference/authentication-in-a-workflow
https://docs.github.com/ja/rest/reference/code-scanning#upload-an-analysis-as-sarif-data